src/components/SlateEditor/plugins/grid/GridForm.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import styled from '@emotion/styled';
+import { ButtonV2 } from '@ndla/button';
+import { spacing } from '@ndla/core';
+import { RadioButtonGroup, GridType } from '@ndla/ui';
+import { Formik, FieldProps } from 'formik';
+import { useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import FormikField from '../../../FormikField';
+import validateFormik, { RulesType } from '../../../formikValidationSchema';
+
+interface GridFormValues {
```

Comment:
Burde du ikke få inn `border` her også?

Reply:
Jeg la til border i starten for å kunne enklere se Grid'en men nå som vi har wrapper cellene så er det uansett lett å se innholdet.